src/vue/Components/Options/Accounts.vue

```diff
@@ -1,10 +1,11 @@
 <template>
     <div>
-        <div class="account-options">
-            <translate tag="label" for="master-account" say="SettingsAccountsMain"/>
-            <select-field id="master-account" v-model="defaultServer" :options="serverOptions"/>
+        <div class="account-options" v-if="serverOptions.length > 2">
+            <div>
+                <translate tag="label" for="master-account" say="SettingsAccountsMain"/><select-field id="master-account" v-model="defaultServer" :options="serverOptions" :required="true" />
+            </div>
         </div>
-        <account-list :servers="servers" v-on:change="loadData"/>
+        <account-list :servers="servers" :defaultServer="defaultServer" v-on:change="loadData"/>
```

Comment:
What does the default server do? I don't see anything related to it in `src/vue/Components/Accounts/AccountList.vue`